src/Symfony/Component/Console/Application.php

```diff
@@ -899,13 +899,13 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
         try {
             $exitCode = $command->run($input, $output);
         } catch (\Exception $e) {
-            $event = new ConsoleTerminateEvent($command, $input, $output, $e->getCode());
-            $this->dispatcher->dispatch(ConsoleEvents::TERMINATE, $event);
+            $terminateEvent = new ConsoleTerminateEvent($command, $input, $output, $e->getCode());
+            $exceptionEvent = new ConsoleExceptionEvent($command, $input, $output, $e, $terminateEvent->getExitCode());
```

Comment:
Well at this point `getExitCode()` would be equal to `$e->getCode()` (it's assigned in the constructor of ConsoleTerminateEvent).

I guess dispatching the event could result in it being modified though, is what you are implying?